_pages/en/it-picture-medium-term.md

```diff
@@ -26,124 +26,131 @@ The foundational pieces for "Government as a service" vision are available and i
+Said time also includes time for informal, self-directed, on-the-job learning and completing courses or attending conferences.
+The budget for learning opportunities has increased every year for the past 5 years.
 
-Employees attend presentations and events, which are held at least once a month by IT and provide opportunities and spaces for employees to learn and share their knowledge, such as a project they are working on or something they are investigating.
-People who attend conferences or take courses then share their learnings via presentations or trip reports.
-IT employees are encourage to complete online courses and form study groups as part of normal work activity.
+Employees attend presentations and events, which are held at least once a month by IT and provide opportunities and spaces for employees to learn and share their knowledge, such as a project they are working on or something they are researching.
+People who attend conferences or take courses share their learning via presentations or trip reports.
+IT employees are encouraged to complete online courses and form study groups as part of normal work activity.
 
-Cooperation between IT and business has increased and there is a conscious effort to break down silos in IT and the wider organization.
-Responsibility for building, deploying and maintaining systems, is shared by cross-functional teams that include representatives from each functional area of the software delivery process (business analysts, developers, quality engineers, operations, security, ..).
+Cooperation between IT and business has increased and there is a conscious effort to break down silos.
+Responsibility for building, deploying and maintaining systems, is shared by cross-functional teams that include representatives from each functional area of the software delivery process (business analysts, developers, quality engineers, operations, security, ...).
 
-Overall employees working in IT at ESDC are happier, more productive and more up to date on technology, making them better able to support the needs of the department so that it can deliver more efficient services to Canadians.
+Overall employees working in IT at ESDC are happier, more productive and more up to date on technology, making them better able to support the evolving needs of the department so that it can deliver more efficient services to Canadians.
 
 #### Process
 
-- https://cloud.google.com/solutions/devops/devops-process-team-experimentation
-- https://cloud.google.com/solutions/devops/devops-process-streamlining-change-approval
-- https://cloud.google.com/solutions/devops/devops-process-customer-feedback
-- https://cloud.google.com/solutions/devops/devops-process-work-visibility-in-value-stream
-- https://cloud.google.com/solutions/devops/devops-process-working-in-small-batches
+- [https://cloud.google.com/solutions/devops/devops-process-team-experimentation](https://cloud.google.com/solutions/devops/devops-process-team-experimentation)
```

Comment:
Same as above, my understanding is that we wanted to embed these in specific lines of the paragraph?